src/modules/Dashboard/features/Participant/Assignments/ActivityListItem/ActivityListItem.tsx

```diff
@@ -3,18 +3,18 @@ import {
   StyledActivityThumbnailImg,
   StyledFlexTopCenter,
 } from 'shared/styles';
-import { ActivityFlowThumbnail } from 'modules/Dashboard/components';
+import { ActivityFlowThumbnail, ActivityStatusChip } from 'modules/Dashboard/components';
 import { FlowChip } from 'shared/components';
 
 import { ActivityListItemProps } from './ActivityListItem.types';
 import { StyledActivityListItem, StyledActivityName } from './ActivityListItem.styles';
 
 export const ActivityListItem = ({
-  activityOrFlow: { isFlow, images, name },
+  activityOrFlow: { isFlow, images, name, status },
   onClick,
   children,
 }: ActivityListItemProps) => (
-  <StyledActivityListItem as="button" onClick={onClick}>
+  <StyledActivityListItem as={onClick ? 'a' : 'div'} onClick={onClick}>
```

Comment:
I don't think `<a>` is the appropriate element to use here — we're passing an `onClick`, not an `href`.

Because this will render an anchor tag without an `href` attribute, which will not receive focus and users who aren't using a pointer-type device will be unable to press on the list item.

I'm _guessing_ you went this route, rather than keeping it as a button, since nesting buttons isn't valid HTML and browsers will try to correct this. Technically, interactive elements inside an anchor tag are _also_ invalid, even if browsers are more forgiving with how they'll render this. The HTML police might not be coming around to investigate, but it's something to keep in mind.

I haven't looked in detail into how you've adapted these into accordions, which likely complicates this further, but I wonder if we should just move the `onClick` from the root node to the first child, which wraps all of the static content on the left, and avoid nesting the buttons passed in as children on the right. This also avoids misclicks — missing a button by a few pixels and navigating to the specific page, rather than opening the action menu, which happened to me at least a few times while testing 😅.

Reply:
I hear ya. I wasn't concerned that the wrapper wasn't accessible because its click action is just a duplicate of the action of the View Data button, which is accessible; avoiding having the container gain keyboard focus makes navigating by keyboard quicker and more intuitive (one less redundant tab press). But I didn't fully clock that issue with nested interactive elements – and in general, I'd like to keep the HTML police on my good side! Especially as violations increase the likelihood of eventually polluting the DevTools console with annoying warnings.

Will come up with a fix!

Reply:
It will be easier to implement this as part of #1943 due to the overlap between the two PRs around card clickability. So I'll merge this as-is so as to avoid a lot of code conflict wrangling.